Sources/StreamChatUI/Appearance+Formatters/PollTimestampFormatter.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// A formatter that converts the poll timestamps to textual representation.
+public protocol PollTimestampFormatter {
+    func formatDay(_ date: Date) -> String
+    func formatTime(_ date: Date) -> String
+}
+
+/// The poll timestamp formatter used in poll votes and comments.
+open class DefaultPollTimestampFormatter: PollTimestampFormatter {
+    /// The formatter to show the day.
+    public var dayFormatter: DateFormatter = {
+        let formatter = DateFormatter()
+        formatter.dateFormat = "dd/mm/YY"
```

Comment:
`setLocalizedDateFormatFromTemplate` otherwise the order of days and months is incorrect for en-US and en-GB